frontend/src/components/SystemSubmitDrawer/index.tsx

```diff
@@ -57,16 +59,38 @@ export function SystemSubmitDrawer(props: Props) {
   const [otherTargetLang, setOtherTargetLang] = useState(false);
 
   const [form] = useForm<FormData>();
-  const { onClose } = props;
+  const { systemToEdit, onClose, ...rest } = props;
+  const editMode = systemToEdit !== undefined;
+
+  const resetAllFormFields = useCallback(() => {
+    form.resetFields([
```

Comment:
I think this solution is fine, but I wonder if it'd also be possible to get all fields automatically without doing it manually, such as [this](https://stackoverflow.com/a/4431223).

Reply:
I looked up antd documentation and they don't provide a way to reset all fields so it seems that listing all the fields is the easiest option.

Reply:
Thanks, I changed it to `form.resetFields()`. I dug a bit deeper, and calling resetFields without providing any arguments resets all fields. It's also used in antd's [official form examples ](https://github.com/ant-design/ant-design/blob/7bd78e7156e0101a6d635b174751b3b686a066ab/components/mentions/demo/form.md).